indigo_api/templates/indigo_api/akn/coverpage_act.html

```diff
@@ -191,11 +191,9 @@ <h2>{{ subt }}</h2>
       {% block publication_document_missing %}
         {% with document.work.place.settings.no_publication_document_text as no_publication_document_text %}
         {% if not document.work.publication_document and no_publication_document_text %}
-          {% block verification_notice %}
-            <li class="verification-notice">
-              [{% trans no_publication_document_text %}]
-            </li>
-          {% endblock %}
+          <li class="verification-notice">
```

Comment:
Why remove the block?

Reply:
Oh there's already the surrounding block, never mind.

Reply:
it isn't overridden anywhere?

Reply:
It *could* be, we don't know if others are overriding this template.